cmd/utils.go

```diff
@@ -70,3 +72,130 @@ func mapCertificatesByTeam(certificates []certificateutil.CertificateInfoModel)
 	}
 	return certificatesByTeam
 }
+
+func getIOSCodeSignGroup(archivePath string, installedCertificates []certificateutil.CertificateInfoModel) (xcarchive.IosArchive, *export.IosCodeSignGroup, error) {
```

Comment:
why is there separate function for getIOSCodeSignGroup and getMacOSCodeSignGroup?